### PR TITLE
chore: 3.12 update

### PR DESCRIPTION
**Pull request by saintsebastian**

### CODE REVIEW

File: config/ember-try.js [hunk 1]
```diff
@@ -2,85 +2,72 @@
...
 const getChannelURL = require('ember-source-channel-url')
 
-module.exports = function() {
-  return Promise.all([
-    getChannelURL('release'),
-    getChannelURL('beta'),
-    getChannelURL('canary')
-  ]).then((urls) => {
-    return {
-      useYarn: true,
```
Comment (romgere) [inline config/ember-try.js]: I had some CI problems with my PR because I forgot this line.

File: package.json [hunk 2]
```diff
@@ -28,39 +28,40 @@
...
-    "ember-resolver": "^5.0.1",
-    "ember-source": "~3.8.0",
-    "ember-source-channel-url": "^1.1.0",
-    "ember-try": "^1.0.0",
+    "ember-qunit": "^4.5.1",
+    "ember-resolver": "^5.3.0",
+    "ember-source": "~3.13.0",
+    "ember-source-channel-url": "^2.0.1",
+    "ember-try": "^1.2.1",
     "eslint-config-peopledoc": "^1.5.0",
```
Comment (romgere) [inline package.json]: Did you plan to update in a separate PR ?

Comment (saintsebastian) [reply package.json]: you are right, might as well be dont now

**Comment by saintsebastian** [comment]
I also removed testing against 2.18, let me know if you disagree with this

**Comment by saintsebastian** [comment]
@romgere thanks for the review, this is now updated

**Comment by romgere** [comment]
LGTM, but I don't understand why this test is still expected : 
image
Is it a repository config issue ?

**Comment by saintsebastian** [comment]
@romgere i don't know either, i removed the job, but maybe CI cache preserved it?

**Comment by njoyard** [comment]
Yes it is most likely a repo config issue, the 2.18 job is registered as a required check. The repo conf should be updated to look for the whole workflow, but IDK who can do that except @xavier

**Comment by mgu** [comment]
I've updated the github settings and remove this test from the list of required tests

**Pull request merged**